src/clj/ataru/applications/automatic_payment_obligation.clj

```diff
@@ -0,0 +1,24 @@
+(ns ataru.applications.automatic-payment-obligation
+  (:require [ataru.background-job.job :as job]
+            [ataru.koodisto.koodisto-codes :as codes]
+            [ataru.applications.application-store :as application-store]))
+
+(defn nationality-finland? [{:keys [kansalaisuus]}]
+  (some true? (map #(= codes/finland-country-code %) kansalaisuus)))
+
+(defn automatic-payment-obligation-job-step
+  [{:keys [person-oid]}
+   {:keys []}]
+  (let [oids (->> (application-store/onr-applications person-oid)
+                  (filter nationality-finland?)
+                  (map :oid))]
+    (for [application (map application-store/get-latest-application-by-key oids)]
+      (for [hakukohde-oid (:hakukohde application)]
+        (application-store/save-application-hakukohde-review
+          (:key application)
+          hakukohde-oid
+          "payment-obligation"
+          "not-obligated"
+          nil
+          false)))
```

Comment:
Jos tän halutaan toimivan samoin kun automaattisen hakukelposuuden, niin pelkkä `ei-upsert` ei riitä, vaan myös `Tarkistamatta` tilasta tulee siirtyä `not-obligated` tilaan.